debian/changelog

```diff
@@ -1,3 +1,9 @@
+daos (2.3.106-2) unstable; urgency=medium
+  [ Tom Nabarro ]
+  * Add numactl requires for server and agent
```

Comment:
Looks like the ticket was updated to just prescribe this for `daos_admin`.

Reply:
updated both change log entries